core/tests/test_utils.py

```diff
@@ -537,11 +590,14 @@ def _parse_json_response(self, json_response, expect_errors=False):
 
         return json.loads(json_response.body[len(feconf.XSSI_PREFIX):])
 
-    def get_json(self, url, params=None, expect_errors=False,
-                 expected_status_int=200):
+    def get_json(self, url, params=None, expected_status_int=200):
         """Get a JSON response, transformed to a Python object."""
+        have_errors = False
```

Comment:
Var name should probably be expect_errors. Currently you are trying to have this variable state whether the test expects an error after the response comes back. It doesn't say whether the response actually has an error or not.

Similar comments might apply elsewhere; please check.

Reply:
done